aioshelly/rpc_device/device.py

```diff
@@ -180,13 +190,13 @@ async def initialize(self, async_init: bool = False) -> None:
             self.initialized = True
         except (MacAddressMismatchError, FirmwareUnsupported) as err:
             self._last_error = err
-            _LOGGER.debug("host %s: error: %r", ip, err)
+            _LOGGER.debug("host %s:%s: error: %r", ip, self.port, err)
```

Comment:
I think we should be consistent here, either we use local variables `ip` and `port` or `self.ip_address` and `self.port` everywhere.